.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,13 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  golangci-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+      - name: golangci-lint
+        uses: reviewdog/action-golangci-lint@v1
+        with:
+          github_token: ${{ secrets.REVIEWDOG_GITHUB_TOKEN }}
```

Comment:
Where is this token kept right now?

Reply:
I have it in 1pass, could use a shared vault maybe?